solver/src/main/java/org/chocosolver/solver/constraints/nary/cumulative/CumulFilter.java

```diff
@@ -56,4 +56,15 @@ public CumulFilter(int nbMaxTasks){
 	 * @throws ContradictionException
 	 */
 	public abstract void filter(IntVar[] s, IntVar[] d, IntVar[] e, IntVar[] h, IntVar capa, ISet tasks, Propagator<IntVar> aCause) throws ContradictionException;
+
+	protected void propStartDurationEndRelation(IntVar[] s, IntVar[] d, IntVar[] e, IntVar[] h, Propagator<IntVar> aCause) throws ContradictionException {
```

Comment:
what if we don't do that at all and leave it to arithm ?

Reply:
As I said, the idea is to avoid a ping-pong between arithm constraints and the cumulative one, as the latter could be long to initiate.  But maybe it is OK to allow it

Reply:
Ok I found a new reason : some filtering algorithms rely on the hypothesis of the task relation, especially the one in NRJCumulFilter

But I still can't really explain why we have the performance issues